x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/kerberos/KerberosRealmAuthenticateFailedTests.java

```diff
@@ -76,7 +76,8 @@ public void testAuthenticateDifferentFailureScenarios() throws LoginException, G
             assertThat(result.getStatus(), is(equalTo(AuthenticationResult.Status.CONTINUE)));
         } else {
             if (validTicket) {
-                final User expectedUser = new User(username, roles.toArray(new String[roles.size()]), null, null, null, true);
+                final User expectedUser = new User(stripRealmName(username), roles.toArray(new String[roles.size()]), null, null, null,
```

Comment:
nit: its much cleaner if you just move the whole `new User(...` statement to a new line